src/libmongoc/tests/unified/operation.c

```diff
@@ -87,23 +88,27 @@ operation_create_change_stream (test_t *test,
       goto done;
    }
 
+   opts = bson_copy (bson_parser_get_extra (parser));
+   if (op->session) {
```

Comment:
Since I'm not familiar with libmongo's test runner, I want to confirm that `op->session` here is the `mongoc_client_session_t` that has already been dereferenced from the "session" entity ID in the operation arguments. Is that correct?

That is my understanding from reading the "Check for a "session" argument in all operations..." section in `operation_run`.

Reply:
That was my understanding as well. 

From the "session" argument parsing logic [here](https://github.com/mongodb/mongo-c-driver/blob/master/src/libmongoc/tests/unified/operation.c#L2321-L2341), it looks like `op->session_id` is set to the "session" entity ID in the operation arguments, and then `op->session` is set to `entity_map_get_session` of that `session_id`.